src/slack-model.jl

```diff
@@ -1,6 +1,29 @@
 export SlackModel, SlackNLSModel
 
-get_slack_ind(jl, ind) = map(b -> findfirst(x -> x == b, ind), intersect(jl, ind))
+function get_slack_ind(jl, ind)
```

Comment:
It returns the indices of `jl` relative to the set `ind`. So, [`2`] within [`2`, `4`] is [`1`]. I added a docstring saying `Return the relative indices of the set of indices jl within the set of indices ind.`

Reply:
It sounds like what `findall` does?

Reply:
Maybe, I am not sure to see how that would work though

Reply:
I think you want `indexin(jl, ind)`. It returns `[findfirst(j .== ind) for j in jl]`. The difference is the `intersect`, which I think you don't need, unless you expect repetition. If that is the case, you can call `unique` on the result.

Reply:
Thanks for the suggestion! I need the interesect because some elements of `jl` might not be in `ind`. I added:
```
get_slack_ind(jl, ind) = findall(!isnothing, indexin(jl, ind))
```
and added the `findall` because `indexin` return a `Vector{Union{Int, Nothing}}`.